requirements/base.in

```diff
@@ -1,7 +1,7 @@
 # Core requirements for using this application
 -c constraints.txt
 
-Django>=1.11                          # Web application framework
+Django>=2.2,<3.0                          # Web application framework
```

Comment:
You should update the constraint to `django<2.3` and move the constraint in the `constraints` file.